corehq/apps/cloudcare/templates/cloudcare/partials/form_entry/entry_file.html

```diff
@@ -2,15 +2,20 @@
 
 <script type="text/html" id="file-entry-ko-template">
   <div class="d-flex">
-    <input type="file" data-bind="
+    <input type="file" style="display: none;" data-bind="
```

Comment:
we try to not use inline style. You could use `class="d-none"` if bootstrap is available here.